clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/ValueCodecFactory.java

```diff
@@ -79,11 +91,11 @@ public T decode(Object input) {
       ValueWrapper data = (ValueWrapper) input;
       ByteArrayInputStream bais = new ByteArrayInputStream(data.getValue());
       try {
-        try (ObjectInputStream ois = new ObjectInputStream(bais)) {
```

Comment:
Pretty sure the assertion at the end of this method should not be needed, since you shouldn't be ignoring the IOException.  If you mean to assert that the IOException itself is impossible then throw the AssertionError from there wrapping the IOException.

Reply:
Removed the uneccessary assertion